packs/feats/force-of-nature.json

```diff
@@ -28,7 +28,23 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
```

Comment:
Didn't add a note on damage received as we can't predicate on silver, so might have been pretty noisy in the chat.